package.json

```diff
@@ -24,7 +24,7 @@
     "build": "node-gyp build",
     "readme": "jsdoc2md --template docs/README.hbs index.js > README.md",
     "lint-js": "eslint *.js",
-    "lint-cpp": "cpplint src/*",
+    "lint-cpp": "cpplint --recursive src/*",
```

Comment:
I'm afraid that will still fail, given that Windows will think `src/*` is literally a file.`cpplint --recursive src` works for me.